src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -69,178 +69,178 @@ public enum MeasurementDisplayStyle
         BarAndNumber
     }
 
-    /// <summary>
-    ///     The convention to be used in labeling computational basis states
-    ///     given their representations as strings of classical bits.
-    /// </summary>
-    [JsonConverter(typeof(StringEnumConverter))]
-    public enum BasisStateLabelingConvention
-    {
-        /// <summary>
-        ///     Label computational states directly by their bit strings.
-        /// </summary>
-        /// <example>
-        ///     Following this convention, the state |0⟩ ⊗ |1⟩ ⊗ |1⟩ is labeled
-        ///     by |011⟩.
-        /// </example>
-        Bitstring,
+    // /// <summary>
```

Comment:
Similarly here, these should be removable now that the relevant code has been ported to qsharp-runtime.